dev/com.ibm.ws.session.cache/resources/com/ibm/ws/session/store/cache/resources/WASSessionCache.nlsprops

```diff
@@ -69,6 +69,10 @@ CONFIG_ATTRIBUTE_IGNORED=SESN0311W: The {0} configuration attribute was specifie
 CONFIG_ATTRIBUTE_IGNORED.explanation=The specified configuration is ignored as it is provided by the specified cacheManagerRef.
 CONFIG_ATTRIBUTE_IGNORED.useraction=No action is required.
 
+UPDATED_CONFIG_NOT_USED_AT_RESTORE=SESN0312W: The HTTP Session Cache configuration attribute {0} has changed when restoring the server process. If the value of the attribute changes when restoring the server process, the application will not use the updated value.
+UPDATED_CONFIG_NOT_USED_AT_RESTORE.explanation=The specified configuration attribute cannot be modified when restoring the server process.
+UPDATED_CONFIG_NOT_USED_AT_RESTORE.useraction=Provide a predetermined value for the configuration attribute before the checkpoint.
```

Comment:
```suggestion
UPDATED_CONFIG_NOT_USED_AT_RESTORE.useraction=To change the value for the attribute, wait until after the server process is restored. Then modify the value and create a new server checkpoint.
```

Reply:
or

Reply:
not sure if this is best practice but we should try to give guidance other than- "dont change the value"